test/XRP/fakes/fake-wallet.ts

```diff
@@ -19,10 +19,10 @@ class FakeWallet extends Wallet {
   /**
    * Initialize a wallet which will always produce the same signature when asked to sign a string.
    *
-   * @param {String} signature The signature this wallet will produce.
+   * @param signature - The signature this wallet will produce.
    */
   constructor(
-    private signature: string,
+    private readonly signature: string,
```

Comment:
Because this is a `private` class member that is never mutated, some rule we have is smart enough to realize that this can be marked `readonly`.

It couldn't do this for a `public` or `protected` member obviously.